forte/pipeline.py

```diff
@@ -466,6 +471,43 @@ def save(self, path: str):
         with open(path, "w", encoding="utf-8") as f:
             yaml.safe_dump(self._dump_to_config(), f)
 
+    def export(self, name: Optional[str] = None) -> Optional[str]:
+        r"""Export pipeline to FORTE_EXPORT_PATH
+        FORTE_EXPORT_PATH is an environment variable
+        The pipeline is saved by :meth:`~forte.pipeline.Pipeline.save`
+        This method will have the following behaviors:
+        - FORTE_EXPORT_PATH will be created if assigned but not found.
+        - If name is not provided, a default one will be used.
+        - When the pipeline is exported multiple time with default name,
+        the export name will be `pipeline-1.yml`, `pipeline-2.yml`...
+        - Raise exception if provided name is taken.
```

Comment:
Docstring suggestions

1. The indent of the list doesn't work properly on the generated doc, see the effect here: https://asyml-forte--690.org.readthedocs.build/en/690/code/pipeline.html#forte.pipeline.Pipeline.export
2. Should also make it explicit that: "If the `name` argument is provided, no suffix will be appended"
3. Should make it explicit what exception will be raised (ValueError in this case)
4. In addition to saying the pipeline saved by `save`, also explain the behavior explicitly: the pipeline will be serialized to a file (and then explain the details of the file
5. Explicitly stated that `FORTE_EXPORT_PATH` should be a directory and all serialized files will be stored here.

Reply:
sry... should have read the CONTRIBUTING.md closely